C/CryptoMiniSat/build_tarballs.jl

```diff
@@ -54,7 +53,7 @@ products = Product[
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("boost_jll"; compat="=1.76.0"),
+    Dependency("boost_jll"),
```

Comment:
Why? Boost ***must*** be pinned: #3024

Reply:
Sorry, I didn't know that (and the link you give doesn't enlighten me further). Are you saying all boosts are binary incompatible? That would give the impression that each package should compile its own boost, otherwise how could two packages with incompatible boosts coexist? Please advise.

Reply:
> Are you saying all boosts are binary incompatible?

Yes, all boost versions are incompatible with each other because they embed the full version number (major.minor.patch) in the soversion.

> That would give the impression that each package should compile its own boost

What does that mean?

> otherwise how could two packages with incompatible boosts coexist? 

Ask the boost developers who the brilliant idea to make each single version ABI-breaking (perhaps they had their own reasons, but that's still a terrible idea in terms of compatibility).

Reply:
OK, got it.
I was suggesting that CryptoMiniSat could download and compile its own boost, and include it. It seems to me that in the current setup lots of package updates will be held back till when each of them gets updated to use the new boost.